backend/internal/gateway/gateway_slots_test.go

```
@@ -0,0 +1,152 @@
+import (
+	"context"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/brantje/llamarack/backend/internal/auth"
+)
+
+func TestClassifySlotsRoutes(t *testing.T) {
+	spec, params, ok := classify(http.MethodGet, "/v1/slots")
```

golangci-lint (2.13.2)

[error] ineffectual assignment to params

(ineffassign)